### DISCUSSION TITLE
.goreleaser overrides

### DISCUSSION THREAD

**Discussion by jportasa** [goreleaser/goreleaser#1671] [Q&A] [answered]
Hi guys!

My .goreleaser needs an override for files inside nfpm files section.:

Centos 5 needs an specific file.

How can I add this override, now I see I only can override depending on if it is rpm, deb, win, but not by OS version.

overrides:
      rpm:
        centos5:  <<<<===== Something like this
          files:
             'SORUCE_FILE': 'DESTINATION_FILE'

Thanks

**Comment by caarlos0** [comment, marked as answer]
you would need to add multiple nfpm configs, one specific for centos5.
